### PR TITLE
Dishes

### PR DESCRIPTION
**Pull request by fabianlundgren**
Changes proposed in this pull request:

Added content to menu.erb to display Our Menu, dishes and price.
Cucumber and rspec tests are passing.

Ready for review!

@AmberWilkie

### CODE REVIEW

File: features/menu_on_menu_page.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+| name  | price |
+| Pizza      | 20    |
+| Kebab      | 30    |
+
+
+Scenario: Visitor is presented with a menu on menu page
+  Given I visit the menu page
+  Then I should see "Our Menu"
+  And I should see "Pizza" with "20"
+  And I should see "Kebab" with "30"
```
Comment (tochman) [inline features/menu_on_menu_page.feature:16]: Please format your code with correct indents. This in a no go on my part.

File: features/step_definitions/test_steps.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  expect(page).to have_content name
+  expect(page).to have_content price
+end
+
+Given(/^the following dishes exist$/) do |table|
+  table.hashes.each do | dish |
+    Dish.create(dish)
+
+end
+end
```
Comment (tochman) [inline features/step_definitions/test_steps.rb]: Same here, please be mindful of code formatting.

File: lib/controller.rb [hunk 3]
```diff
@@ -54,10 +54,15 @@ class SlowFood < Sinatra::Base
...
-    erb :login
+  get '/menu' do
+    @dishes = Dish.all
+    erb :menu
   end
 
+  # get '/auth/login' do
+  #   erb :login
+  # end
+
```
Comment (tochman) [inline lib/controller.rb]: If you won't be using this method, please remove it. Not cool to keep comments around.

File: lib/views/login.erb [hunk 4]
```diff
@@ -1,5 +1,7 @@
+<!-- <!doctype html>
+
 <form action="/auth/login" method="post">
   <p>Username: <input type="text" name="user[username]" /></p>
   <p>Password: <input type="password" name="user[password]" /></p>
   <input type="submit" value="Log In" />
-</form>
\ No newline at end of file
+</form> -->
```
Comment (tochman) [inline lib/views/login.erb]: Same thing here. If you don't use this view template, please delete it.

File: lib/views/menu.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<h1>Our Menu</h1>
+
+
+<% @dishes.each do | dish | %>
+
+<p>
+<%= dish.name %>, <%= dish.price %>
+</p>
+
+<% end %>
```
Comment (tochman) [inline lib/views/menu.erb]: Please be mindful of empty spaces and code formatting.

File: spec/spec_helper.rb [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (tochman) [inline spec/spec_helper.rb]: I know that this is auto generated code but I would encourage you to remove the comments.

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/step_definitions/test_steps.rb [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+Then(/^I should see "([^"]*)" with "([^"]*)"$/) do |name, price|
+  expect(page).to have_content name
+  expect(page).to have_content price
+end
+
+Given(/^the following dishes exist$/) do |table|
+  table.hashes.each do | dish |
+  Dish.create(dish)
+end
+end
```
Comment (AmberWilkie) [inline features/step_definitions/test_steps.rb]: Formatting here is a problem - if you ever have two `end` statements stacked on each other like this, it means there is an indentation problem. This block should look like:
```
Given(/^the following dishes exist$/) do |table|
  table.hashes.each do | dish |
    Dish.create(dish)
  end
end
```
See how that demonstrates that the `Dish.create(dish)` is *inside* the `table.hashes` block? That's what we mean about "readability"

File: lib/controller.rb [hunk 8]
```diff
@@ -54,6 +54,15 @@ class SlowFood < Sinatra::Base
...
+    erb :menu
+  end
+
+  get '/auth/login' do
+    erb :login
+  end
+
   get '/auth/login' do
     erb :login
   end
```
Comment (AmberWilkie) [inline lib/controller.rb]: You should only have one `get '/auth/login' block.

**Pull request merged**